--- first.sh	2018-12-14 10:34:58.000000000 +0100
+++ new_first.sh	2020-02-05 19:15:33.804166636 +0100
@@ -4,6 +4,7 @@
 #                                                                                                                                                                                                                                            
 # Copyright Werum Software & Systems AG 2011                                                                                                                                                                                                 
 #                                                                                                                                                                                                                                            
+2>/dev/null mv $1 $1.tmp || { echo '[WARNING] first.sh: Exiting because first.sh is already running. Please have a look at the TaskTable!' && exit; } && sleep 10 && xsltproc /usr/local/components/IPFSimulator/etc/removeStats.xsl $1.tmp >$1 && rm -f $1.tmp
                                                                                                                                                                             
 JAVA_HOME_PIC=/usr/local/COTS/pic/java1.8               
 if [ $# == 2 ];then
